### DISCUSSION TITLE
Adding custom label to a metric

### DISCUSSION THREAD

**Discussion by freeyssu** [prometheus/snmp_exporter#1057] [Q&A] [answered]
Hi team.

Can we have a way to add a custom label to a metric?
For example, I would like to add switch model as a label(model=Cisco-1234) of a specific metric.
It seems like there is the only the lookup feature supports by index order, isn't it?

**Comment by SuperQ** [comment, marked as answer]
No, this is done in the Prometheus config.

**Reply by freeyssu** [reply]
ah, I got it :)
Thanks !
